client/src/components/Header.css

```diff
@@ -0,0 +1,32 @@
+@import url('https://fonts.googleapis.com/css2?family=Oswald:wght@700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@700&display=swap');
+
+header {
+    background-color: #ffffff;
+    height: 72px;
+}
+
+a, a:visited {
+    color: #000000;
+    text-decoration: none;
+}
+
+.header-logo_container {
+    display: flex;
+    align-items: flex-end;
+    flex-shrink: 0;
+    gap: 34px;
+    margin-left: 82px;
+}
+
+.header-text {
+    font-family: 'Oswald', 'Roboto', sans-serif;
```

Comment:
Header text should just be Oswald or sans-serif since it's the decodenoir logo font

Reply:
removed